lib/util/common.js

```diff
@@ -49,10 +62,12 @@ function addHook (fastify, pluginOptions) {
     routes,
     Ref () {
       const externalSchemas = cloner(Array.from(sharedSchemasMap.values()))
-      return Ref(Object.assign(
-        { applicationUri: 'todo.com' },
-        pluginOptions.refResolver,
-        { clone: true, externalSchemas })
+      return Ref(
+        Object.assign(
+          { applicationUri: 'todo.com' },
+          pluginOptions.refResolver,
+          { clone: true, externalSchemas }
+        )
```

Comment:
Did you leave something like Prettier on? Style has changed but not the content.

Reply:
Yes, I prettiered, then the `npm run lint:fix` changed it to this.

I no manually pick my changes to avoid this 👍